components/compliance-service/scripts/grpc.sh

```diff
@@ -24,16 +25,16 @@ function cleanup() {
 trap cleanup EXIT
 
 for i in $(find components/compliance-service -name '*.proto') ; do
+  printf 'GEN: %s\n' "${i}"
```

Comment:
I moved this before the compilation step so that if there are warnings or errors its easier to tell which file was responsible.